R/tm_g_km.R

```diff
@@ -740,13 +762,10 @@ srv_g_km <- function(id,
       anl <- qenv[["ANL"]] # nolint
       teal::validate_has_data(anl, 2)
 
-      input_xticks <- gsub(";", ",", trimws(input$xticks)) %>%
-        strsplit(",") %>%
-        unlist() %>%
-        as.numeric()
-
-      if (length(input_xticks) == 0) {
-        input_xticks <- NULL
+      input_xticks <- if (!is.null(input$xticks)) {
+        as_numeric_from_comma_sep_str(input$xticks, sep = ";")
+      } else {
+        NULL
```

Comment:
`else` not needed as `if` would return (invisible) `NULL` anyway
```suggestion
      input_xticks <- if (!is.null(input$xticks)) {
        as_numeric_from_comma_sep_str(input$xticks, sep = ";")
      }
```